bluetooth-toy-bb8/index.html

```diff
@@ -264,7 +264,7 @@
           if (controlCharacteristic == null) {
             navigator.bluetooth.requestDevice({
```

Comment:
Be sure to either request '22bb746f-2bb0-7554-2d6f-726568705327' as an optional service, or include it in a second filter.